src/config.js

```diff
@@ -43,7 +44,10 @@ export type Config = {
   arch: string,             // CPU architecture
   platform: string,         // operating system
   windows: boolean,         // is windows OS
-  _version: '1'             // config schema version
+  _version: '1',             // config schema version
+  userConfig: UserConfig,
```

Comment:
@KarateCowboy I would not expose `userConfig` here, it adds another redundant way to access `skipAnalytics` and consumers of this should not have to think about whether something is exposed via the user config file or not.